types/app.d.ts

```diff
@@ -143,6 +144,18 @@ export function eventGrid(name: string, options: EventGridFunctionOptions): void
  */
 export function cosmosDB(name: string, options: CosmosDBFunctionOptions): void;
 
+/**
```

Comment:
> Could you tell me what happens when you try to deploy that using the Consumption plan? It'd be nice if there were a nice error message

@charlesgardyn this npm package doesn't have visibility into what plan its running under, so unfortunately I don't think it's possible for us to do anything. For the record it would be at most a warning, since your app needs to be able to run without the warmup trigger regardless of plan type. The [functions host](https://github.com/Azure/azure-functions-host) might be able to do the warning, but I don't see warmup triggers used often enough that I feel like bothering them

Reply:
I see, I thought that was possible. It's ok if not, it's for some specific use cases indeed